tests/simple-3-color-test.toit

```diff
@@ -4,21 +4,24 @@
 
 import png-display show *
 import pixel-display show *
+import pixel-display.element show *
```

Comment:
Element and style are so essential that they should be exported from `pixel-display`.